test/test_protein_sequences.py

```diff
@@ -96,10 +96,12 @@ def make_dummy_protein_sequence(
 
     return ProteinSequence(
         translations=[translation] * n_translations,
-        supporting_variant_reads=[None] * n_supporting_variant_reads,
-        total_variant_reads=n_total_variant_reads,
-        supporting_transcripts=[None] * n_supporting_reference_transcripts,
-        total_transcripts=n_total_reference_transcripts,
+        overlapping_reads=[None] * n_supporting_variant_reads,
```

Comment:
That's actually a mistake, the number of alt reads should be n_total_variant_reads